frontend/packages/process-editor/src/components/Canvas/Canvas.module.css

```
@@ -15,7 +19,35 @@
   font-weight: 500;
 }
 
-.alertContainer  {
+.alertContainer {
   composes: container;
   max-width: 780px;
-}
\ No newline at end of file
+}
+
+div[class*='bpmn-icon-task-generic'] {
+  background-color: black;
+}
+
+div[class*='bpmn-icon-task-generic']:hover {
+  background-color: #0086e6;
+}
+
+div[class*='bpmn-icon-data-task'] {
+  -webkit-mask: url('../../../img/DataTask.svg') 0 0 /46px 46px;
+  mask: url('../../../img/DataTask.svg') 0 0 /46px 46px;
+}
+
+div[class*='bpmn-icon-confirmation-task'] {
+  -webkit-mask: url('../../../img/ConfirmationTask.svg') 0 0/46px 46px;
+  mask: url('../../../img/ConfirmationTask.svg') 0 0/46px 46px;
+}
+
+div[class*='bpmn-icon-feedback-task'] {
+  -webkit-mask: url('../../../img/FeedbackTask.svg') 0 0/46px 46px;
+  mask: url('../../../img/FeedbackTask.svg') 0 0/46px 46px;
+}
+
+div[class*='bpmn-icon-signing-task'] {
+  -webkit-mask: url('../../../img/SignTask.svg') 0 0/46px 46px;
+  mask: url('../../../img/SignTask.svg') 0 0/46px 46px;
+}
```

What do you think about centralising the mask-icon-size into a reusable variable? See the example below. 😊 

```suggestion
:root {
  --mask-icon-size: 0 0/46px 46px;
}

div[class*='bpmn-icon-data-task'] {
  -webkit-mask: url('../../../img/DataTask.svg') var(--mask-icon-size);
  mask: url('../../../img/DataTask.svg') var(--mask-icon-size);
}

div[class*='bpmn-icon-confirmation-task'] {
  -webkit-mask: url('../../../img/ConfirmationTask.svg') var(--mask-icon-size);
  mask: url('../../../img/ConfirmationTask.svg') var(--mask-icon-size);
}

div[class*='bpmn-icon-feedback-task'] {
  -webkit-mask: url('../../../img/FeedbackTask.svg') var(--mask-icon-size);
  mask: url('../../../img/FeedbackTask.svg') var(--mask-icon-size);
}

div[class*='bpmn-icon-signing-task'] {
  -webkit-mask: url('../../../img/SignTask.svg') var(--mask-icon-size);
  mask: url('../../../img/SignTask.svg') var(--mask-icon-size);
}
```

---

Great suggestion!